provisioning/containers/gui-common/websockify/novnc-overrides/app/ui.js

```diff
@@ -9,26 +9,240 @@ const ois = UI.initSettings;
 UI.initSettings = () => {
   ois();
   if (window.websockifyTargetUrl) {
-    UI.forceSetting('path', window.websockifyTargetUrl);
+    UI.forceSetting('path', `${window.websockifyTargetUrl}`);
   } else {
     let { pathname } = window.location;
-    pathname = pathname.split('/').filter(e => e).join('/');
+    pathname = pathname
+      .split('/')
+      .filter((e) => e)
+      .join('/');
     if (pathname) UI.forceSetting('path', pathname + '/vnc');
   }
   UI.forceSetting('show_dot', true);
   UI.forceSetting('reconnect', true);
   UI.forceSetting('reconnect_delay', Math.floor(Math.random() * 2000));
   window.onhashchange();
-}
+};
```

Comment:
?

Reply:
The automatic formatter did that, sorry 😮‍💨